composer.json

```diff
@@ -16,7 +16,8 @@
     },
     "suggest": {
         "dflydev/markdown": "~1.0",
-        "erusev/parsedown": "~1.0"
+        "erusev/parsedown": "~1.0",
+        "league/commonmark": "*"
```

Comment:
I completely agree with your point, but I think it'll be okay in this specific case since the `CommonMarkConverter` class should **never** ever change, even between major versions.

That being said, I wouldn't be opposed to changing it.